src/common/components/paymentMethods/coverFees/coverFees.component.js

```diff
@@ -20,16 +24,16 @@ class CoverFeesController {
   }
 
   $onInit () {
+    this.$rootScope.$on(brandedCheckoutAmountUpdatedEvent, () => {
+      this.feesCalculated = false
+      this.$onInit()
```

Comment:
Will calling this again add another event listener that will start calling this multiple times?

Reply:
Hmm, possibly. It might be getting called twice. I think I started with it in the constructor, but something wasn't working. I might try that again.